src/composables/useStorageLocal.ts

```diff
@@ -0,0 +1,30 @@
+import { storage } from 'webextension-polyfill'
+import {
+  useStorageAsync,
+  StorageLikeAsync,
+  MaybeRef,
+  StorageAsyncOptions,
+  RemovableRef,
+} from '@vueuse/core'
+
+const storageLocal: StorageLikeAsync = {
+  removeItem(key: string) {
+    return storage.local.remove(key)
+  },
+
+  setItem(key: string, value: string) {
+    return storage.local.set({ [key]: value })
+  },
+
+  async getItem(key: string) {
+    return (await storage.local.get(key))[key]
+  },
+}
+
+const useStorageLocal = <T>(
```

Comment:
```suggestion
export const useStorageLocal = <T>(
```

Let's use named exports instead of default